### PR TITLE
removing unused variables in zipkin DoPostAsync method

### LINKED ISSUE #489: ZipkinTraceExporter DoPostAsync method
I was going through OpenTelemetry code checking some implementation details and found this:

```csharp
private async Task DoPostAsync(HttpClient client, HttpRequestMessage request)
{
      using (var response = await client.SendAsync(request).ConfigureAwait(false))
      {
            if (response.StatusCode != HttpStatusCode.OK &&
                response.StatusCode != HttpStatusCode.Accepted)
            {
                 var statusCode = (int)response.StatusCode;
            }
      }
}
```
It seems that both the `if` statement and the `var` declaration are done but don't add any value and could be removed. Is my understanding correct? If so I am more than willing to remove it.

### PR DESCRIPTION
**Pull request by eddynaka**
Fixes #489

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
@eddynaka please sign off CLA. Thank you for contribution!

**Comment by eddynaka** [comment]
@SergeyKanzhelev , just signed. Can you check? Thanks!

File: src/OpenTelemetry.Exporter.Zipkin/ZipkinTraceExporter.cs [hunk 1]
```diff
@@ -127,11 +127,7 @@ private async Task DoPostAsync(HttpClient client, HttpRequestMessage request)
         {
             using (var response = await client.SendAsync(request).ConfigureAwait(false))
```
Comment (MikeGoldsmith) [inline src/OpenTelemetry.Exporter.Zipkin/ZipkinTraceExporter.cs]: We can do away with assigning the response here too.

**Comment by MikeGoldsmith** [comment]
@eddynaka CLA check has passed now. I've made one suggestion.

File: src/OpenTelemetry.Exporter.Zipkin/ZipkinTraceExporter.cs (hunk 1 above)
Comment (eddynaka) [reply src/OpenTelemetry.Exporter.Zipkin/ZipkinTraceExporter.cs]: Hi @MikeGoldsmith , your suggestion would be this:
```csharp
using (await client.SendAsync(request).ConfigureAwait(false))
{
     // disposing response}
```

**Comment by eddynaka** [comment]
@MikeGoldsmith just updated! Thank you!

File: src/OpenTelemetry.Exporter.Zipkin/ZipkinTraceExporter.cs (hunk 1 above)
Comment (MikeGoldsmith) [reply src/OpenTelemetry.Exporter.Zipkin/ZipkinTraceExporter.cs]: I don't think we need the using statement either.

For example:
```suggestion
await client.SendAsync(request).ConfigureAwait(false);
```

Comment (eddynaka) [reply src/OpenTelemetry.Exporter.Zipkin/ZipkinTraceExporter.cs]: just updated!

**Pull request merged**